src/index.ts

```diff
@@ -1,9 +1,5 @@
-export * from "./client/index"
```

Comment:
with these imports being removed shouldn't the actual files also be deleted?

Reply:
line 3 to 7 is still been used

Reply:
or if you're referring to the file they are referring, they have been deleted in the previous commit